feature-detects/css/calc.js

```diff
@@ -21,7 +21,7 @@ define(['Modernizr', 'createElement', 'prefixes'], function( Modernizr, createEl
   Modernizr.addTest('csscalc', function() {
     var prop = 'width:';
     var value = 'calc(10px);';
-    var el = createElement('div');
+    var el = createElement('a');
```

Comment:
No, changing because in IE 10+, non valid elements created inside of an SVG
context don't have a style attribute. None of the caniuse mappings broke,
so it should be golden. I can hand diff this and master in our browsers to
be sure, however

On Mon, Apr 20, 2015, 7:31 PM Ryan Seddon <notifications@github.com> wrote:

> In feature-detects/css/calc.js
> <https://github.com/Modernizr/Modernizr/pull/1583#discussion_r28738838>:
>
> > @@ -21,7 +21,7 @@ define(['Modernizr', 'createElement', 'prefixes'], function( Modernizr, createEl
> >    Modernizr.addTest('csscalc', function() {
> >      var prop = 'width:';
> >      var value = 'calc(10px);';
> > -    var el = createElement('div');
> > +    var el = createElement('a');
>
> Are you just changing div to be a because it's shorter? What about the
> fact that anchors are inline and divs are block this may affect some tests?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Modernizr/Modernizr/pull/1583/files#r28738838>.
>